types/groupMember.js

```diff
@@ -0,0 +1,38 @@
+const Capabilities = require('./capabilities')
+const Serializable = require('./serializable')
+
+class GroupMember extends Serializable {
+  constructor(clientID, capabilities) {
+    super()
+    this.clientID = clientID
+    this.membershipKey = null
```

Comment:
Any reason to not init the membership key here?

Reply:
Sorry I forget, did we have a discussion outside of why this shouldn't be part of the init?